src/main/kotlin/com/cognifide/gradle/aem/pkg/ComposeTask.kt

```diff
@@ -28,24 +28,27 @@ open class ComposeTask : Zip(), AemTask {
     @Nested
     final override val config = AemConfig.of(project)
 
-    @InputFiles
-    val vaultFilters = mutableListOf<File>()
-
     @InputDirectory
     val vaultDir = AemTask.temporaryDir(project, PrepareTask.NAME, AemPackagePlugin.VLT_PATH)
 
+    @Input
+    val filterRoots = mutableSetOf<String>()
+
     @Internal
-    private var bundleCollectors: List<() -> Unit> = mutableListOf()
+    val filterRootDefault = { subproject: Project, config: AemConfig ->
```

Comment:
this is for subprojects that don't have filter.xml file at all. now default filter could be customized

for instance:
```
aemCompose {
    filterRootDefault = { subproject, config -> "<filter root=\"${config.bundlePath}\" mode=\"update\"/>" }
}
```